src/pygama/lgdo/lh5_store.py

```diff
@@ -937,7 +937,10 @@ def load_nda(
                 data, _ = sto.read_object(f"{lh5_group}/{par}", f, idx=idx_list[ii])
             if not data:
                 continue
-            par_data[par].append(data.nda)
+            if isinstance(data, WaveformTable):  # HACK / FIXME
```

Comment:
Is this hack necessary? I thought one should be able to put 'waveform/values' in par_list to get this nda...

Reply:
#319

Reply:
Thanks @jasondet  Reverted the chang